Jamulus.pro

```diff
@@ -308,11 +308,17 @@ win32 {
     }
     APPSDIR = $$absolute_path($$APPSDIR, $$PREFIX)
     desktop.path = $$APPSDIR
+    QMAKE_SUBSTITUTES += distributions/jamulus.desktop.in
     desktop.files = distributions/jamulus.desktop
-    # the .desktop file assumes the binary is called jamulus
-    contains(CONFIG, "noupcasename") {
-        INSTALLS += desktop
+    INSTALLS += desktop
```

Comment:
Why not add the desktop and icons in one step?

Reply:
OK!